docs/UsersGuide.asciidoc

```diff
@@ -1134,10 +1151,9 @@ This section provides an overall description of the cleanup strategy and
 how to configure the limits. Cleanup-related parameter for the REST API can
 be found in the <<UsersGuide.asciidoc#_asset_handling,Asset handling>> section under
 <<UsersGuide.asciidoc#_use_of_the_rest_api,Use of the REST API>>.
+For when cleanup happens see <<UsersGuide.asciidoc#_timers_and_triggers,Timers and triggers>>.
```

Comment:
This reference would now make more sense in the "Cleanup" section under "Advanced configuration".